sqlmesh/core/context.py

```diff
@@ -1164,8 +1164,19 @@ def _load_configs(
 
     def _run_janitor(self) -> None:
         expired_environments = self.state_sync.delete_expired_environments()
+        expired_schemas: t.Set[str] = set()
         for expired_environment in expired_environments:
             self.snapshot_evaluator.demote(expired_environment.snapshots, expired_environment.name)
+            expired_schemas.update(
+                {
+                    exp.to_table(
+                        snapshot.qualified_view_name.for_environment(expired_environment.name)
```

Comment:
can't this just be schema_for_environment?

Reply:
It is a `SnapshotTableInfo` and not a `Snapshot`.

Reply:
snapshot.qualified_view_name.schema_for_environment(env)

instead of to_table(...).db ?

Reply:
Oh nm I see now on qualified view